### PR TITLE
Config: Remove MAC addresses user input

### PR DESCRIPTION
**Pull request by orelmisan**
Currently, the user can set two MAC addresses for the traffic gen, and another two for the VMI under test.

In case there is no user input, these four MAC addresses are randomly generated.

This input controls a pretty low-level implementation detail.
Remove the option for the user to set the four MAC addresses.

The four MAC addresses are now always randomly generated.

### CODE REVIEW

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/8700181c14cc24c586e651261dd2405c0066e86a..d306bff945d9d13e55bb7615a3ab07a231ad7c01): Fixed a typo in a commit message.

**Comment by orelmisan** [comment]
Tested against an OpenShift Virtualization 4.14 cluster.

**Comment by orelmisan** [comment]
[Change](https://github.com/kiagnose/kubevirt-dpdk-checkup/compare/d306bff945d9d13e55bb7615a3ab07a231ad7c01..754fe47b30bb6eef67d542c45217ee95f97d7819): Keep the print of the generated MAC addresses.

File: pkg/mainflow.go [hunk 1]
```diff
@@ -70,12 +70,12 @@ func printConfig(checkupConfig config.Config) {
 	log.Printf("%q: %q", config.TrafficGeneratorImageParamName, checkupConfig.TrafficGeneratorImage)
 	log.Printf("%q: %q", config.TrafficGeneratorNodeLabelSelectorParamName, checkupConfig.TrafficGeneratorNodeLabelSelector)
 	log.Printf("%q: %q", config.TrafficGeneratorPacketsPerSecondParamName, checkupConfig.TrafficGeneratorPacketsPerSecond)
-	log.Printf("%q: %q", config.TrafficGeneratorEastMacAddressParamName, checkupConfig.TrafficGeneratorEastMacAddress)
-	log.Printf("%q: %q", config.TrafficGeneratorWestMacAddressParamName, checkupConfig.TrafficGeneratorWestMacAddress)
+	log.Printf("%q: %q", "TrafficGeneratorEastMacAddress", checkupConfig.TrafficGeneratorEastMacAddress)
```
Comment (RamLavi) [inline pkg/mainflow.go:73]: Why don't we keep he const?

Comment (RamLavi) [reply pkg/mainflow.go:73]: Couldn't hurt to keep them, but I won't insist.

**Review by RamLavi — APPROVED** [review]
/lgtm
/approve

**Pull request merged**